src/components/DisplayInput.js

```diff
@@ -33,11 +35,16 @@ class DisplayInput extends React.Component {
     valid: true
   };
 
-  _isLargeOrMediumWindowSize = () => {
+  componentWillMount () {
+    this._labelId = _uniqueId('DI');
+    this._inputId = _uniqueId('DI');
```

Comment:
It's short for `DisplayInput`. Do you think we should change it or add a comment?